### DISCUSSION TITLE
Application page redirects to the same page

### DISCUSSION THREAD

**Discussion by Maxix25** [community/community#128700] [GitHub Education] [answered]
### Hi everyone,

Hey, I'm trying to sign up but when I submit the proof of my academic status, it redirects me to the same page and doesn't say anything, I tried using the brave browser and the firefox browser freshly installed, what can I do?

**Comment by Yatharth-Nanda** [comment]
Im facing the same issue , can someone please answer !

**Reply by 69Youssef** [reply]
go to this link https://education.github.com/discount_requests/application it should work

**Comment by 69Youssef** [comment]
go to this link https://education.github.com/discount_requests/application it should work

**Reply by javawolfpack** [reply]
It doesn't... Seems to be a wide spread issue as seen now 4-5 posts here that I'm commenting on as it just keeps redirecting back to that page when I submit there. Which is the issue that I think is being brought up here.

I have been trying to find a contact at GitHub still who might be able to get an actual person to fix it.

**Comment by SwastikSharma-Dev** [comment]
Facing the Same Issue. If you have found any solution, please help! Or if anyone could help, please!

**Reply by javawolfpack** [reply]
I may have solved this with the developer console in Chrome... both disabled cache as think needed to reload a script file, but then got errors in the console that weren't displayed that my file sizes were wrong.  But did upload my proof finally and say I was verified (though having issues now that not sure the verification is registered correctly on the backend I'm working with GitHub to solve).

**Reply by SwastikSharma-Dev** [reply]
OKAY. Have you got your verification done now with that or not?

**Reply by BartiSzabolcs** [reply]
I have used another, freshly installed browser, it worked.

**Comment by Kota412** [comment]
Yeah, I am facing the same issue. Why can I not just upload the file rather than take an impossible picture with only the front-facing camera? This being the developer's utopia it is supposed to be, you would think these issues would be minimal.

**Reply by BartiSzabolcs** [reply]
The same issue here too. I do not understand why they removed the option to upload.

**Comment by Balu1412** [comment]
Same problem…

**Comment by Maxix25** [comment, marked as answer]
My solution was to use the camera of my phone/laptop/pc and use the builtin camera function that has the form to take the picture and upload it
